src/components/containers/seo.js

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Helmet } from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
-import { LocaleContext } from '../localization'
+import { LocaleContext, Localize } from '../localization'
 import language_config from '../../../i18n-config'
-import TradingImage from 'images/common/practice.png'
+import TradingImage from 'images/common/og_deriv.png'
 
 const non_localized_links = ['/careers', '/careers/']
```

Comment:
Same value in this array. We can add this in the constants and import from there.

Reply:
We can refactor this line Number 9 in the next phase as discussed.